.eslintrc

```diff
@@ -29,6 +27,8 @@ rules:
   no-unreachable: 2
   ## require valid typeof compared string like typeof foo === 'strnig'
   valid-typeof: 2
+  ## disallow use of undefined variables (globals)
+  no-undef: 2
```

Comment:
no-undef is not `Possible Errors` category,
`Variables` category, https://github.com/eslint/eslint/tree/master/docs/rules#variables